ejb/src/main/java/org/wildfly/httpclient/ejb/HttpDiscoveryHandler.java

```diff
@@ -39,14 +36,22 @@
 import java.util.concurrent.ExecutorService;
 
 /**
+ * Http handler for discovery requests.
+ *
  * @author <a href="mailto:tadamski@redhat.com">Tomasz Adamski</a>
  */
 
 public class HttpDiscoveryHandler extends RemoteHTTPHandler {
 
     private final Set<EJBModuleIdentifier> availableModules = new HashSet<>();
+    private final HttpServiceConfig httpServiceConfig;
 
+    @Deprecated
     public HttpDiscoveryHandler(ExecutorService executorService, Association association) {
```

Comment:
this is deprecated instead of deleted just because this class is public (it shouldn't be public in my opinion) and the module is not marked as private-api in WildFly